pyproject.toml

```diff
@@ -19,9 +19,43 @@ repository = "https://github.com/gis-ops/py-osrm"
 [tool.scikit-build]
 # Protect the configuration against future changes in scikit-build-core
 minimum-version = "0.4"
-
 # Setuptools-style build caching in a local directory
 build-dir = "build/{wheel_tag}"
-
 # Build stable ABI wheels for CPython 3.12+
 wheel.py-api = "cp312"
+
+[tool.cibuildwheel]
+test-command = "pytest"
+skip = "*musllinux*"
+archs = "native"
+manylinux-x86_64-image = "ghcr.io/gis-ops/manylinux_2_28_x86_64_py-osrm"
+
+[tool.cibuildwheel.linux]
+build = "cp38-*"
+test-requires = "pytest"
+test-command = "pytest {project}/tests"
+repair-wheel-command = "LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/usr/local/lib/intel64/gcc4.8/ auditwheel repair -w {dest_dir} {wheel}"
+before-test = "osrm-datastore ./tests/test_data/ch/monaco"
+before-all = """
```

Comment:
all this should be done in the `manylinux` image no?

Reply:
I tried to minimize the amount of dependencies done on the `manylinux` side because I thought it might make it more convenient to maneuver around when `osrm-backend` changes in the future. 
But I suppose having `manylinux` build on `osrm-backend` push might take care of that anyways!

Reply:
Yeah we can leave OSRM in here for now, but def we should install the dependencies inside the image.

Reply:
I ended up moving osrm-backend over to the `manylinux` side entirely, since with Conan, the only thing that would've been left over there is the TBB install - and because as discussed, osrm-backend won't change too frequently.